src/config/platformDependentStatusBar/PlatformDependentStatusBar.tsx

```diff
@@ -0,0 +1,27 @@
+import { useContext } from 'react';
+import { View, StatusBar } from 'react-native';
+
+import SettingsContext from '@config/SettingsContext';
+
+import { isIos } from '@constants';
+import ThemedStyles from './styles';
+
+const PlatformDependentStatusBar = () => {
+  const { theme } = useContext(SettingsContext);
+
+  const styles = ThemedStyles(theme);
+
+  if (!isIos) {
```

Comment:
mention the reason of not using StatusBar in iOS

Reply:
same as above